package.json

```
@@ -65,7 +65,7 @@
   "dependencies": {
     "@eslint-community/eslint-utils": "^4.2.0",
     "@eslint-community/regexpp": "^4.6.1",
-    "@eslint/eslintrc": "^2.1.4",
+    "@eslint/eslintrc": "github:eslint/eslintrc#eslint-issue14709",
```

```suggestion
    "@eslint/eslintrc": "^3.0.0",
```

When we release `@eslint/eslintrc` v3.